### PR TITLE
wgetpaste.spec: Add spec file for building RPMs

### PR DESCRIPTION
**Pull request by Philantrop**

### CODE REVIEW

File: wgetpaste.spec [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+%setup -q
+
+%build
+
+%install
+rm -rf $RPM_BUILD_ROOT
+mkdir -p %{buildroot}/%{_bindir}
+
+install -m 0755 %{name} %{buildroot}/%{_bindir}/%{name}
+sed -i -e "s:/usr/bin/env bash:/bin/bash:" %{buildroot}/%{_bindir}/%{name}
```
Comment (Cogitri) [inline wgetpaste.spec]: Is this something we should adopt upstream?

**Comment by Cogitri** [comment]
Another thing, should we really put this in the repository root and not in some folder where we put other packaging related files?

**Comment by Philantrop** [comment]
Sure, I can put the spec file anywhere. I don't see any reason not to accept this, though. It's common practice to include such stuff.

**Comment by Cogitri** [comment]
Alright, wasn't sure what's the common way to do this for Specfiles :)

**Pull request merged**